### PR TITLE
Feature/add warning log function

### PR DESCRIPTION
**Pull request by santiagonoguez**
As we are starting to use warning messages ( as opposed to errors ), I created a reusable function for that.

### CODE REVIEW

File: logger.js [hunk 1]
```diff
@@ -91,6 +91,12 @@ module.exports = (externalLogger, logFolder, moduleName = "unknownmodule")=> {
       if (externalLogger) {externalLogger.log("error", detail, table, moduleName);}
       if (validUiWindow()) {uiWindow.send("errorMessage", userFriendlyMessage || detail);}
     },
+    warning(detail, userFriendlyMessage, table) {
```
Comment (andrecardoso) [inline logger.js]: I'm always confused with this method signature. If I have some json object as details and some string as event,  how is this going to be used? 

```javascript
logger.warning('file not found', JSON.stringify({filePath, operation, etc})); 
```

or 

```javascript
logger.warning(JSON.stringify({filePath, operation, etc}), 'file not found'); 
```

Comment (santiagonoguez) [reply logger.js]: @andrecardoso the second one is correct. detail is more technical, userFriendlyMessage is more 'user' oriented. Also detail goes to BQ, while userFriendly message not ( in the current implementation ).

Comment (tejohnso) [inline logger.js]: Can we get rid of userFriendlyMessage here? For the file it should just log "warning" and the warning string

Comment (santiagonoguez) [reply logger.js]: @tejohnso I've just removed userFriendlyMessage argument on the warning() function.

**Pull request merged**